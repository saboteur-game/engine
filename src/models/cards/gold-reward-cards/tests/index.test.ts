import GoldRewardCard from "..";

describe("GoldRewardCard", () => {
  let goldRewardCard: GoldRewardCard;

  beforeEach(() => {
    goldRewardCard = new GoldRewardCard(2);
  });

  it("can be serialized", () => {
    expect(goldRewardCard.toJSON()).toMatchSnapshot();
  });
});
