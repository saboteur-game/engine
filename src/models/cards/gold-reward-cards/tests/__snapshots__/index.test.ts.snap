// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoldRewardCard can be serialized 1`] = `
Object {
  "id": "test-id-0",
  "value": 2,
}
`;
